docs/standard/components.md

```diff
@@ -14,67 +14,69 @@ ms.assetid: 2e38e9d9-8284-46ee-a15f-199adc4f26f4
 
 # .NET Architectural Components
 
-.NET is made up of a number of key components.  It has a standard library, called the .NET Standard, which is a large set of APIs which runs everywhere.  This standard library is implemented by three .NET runtimes - .NET Framework, .NET Core, and Mono for Xamarin.  The .NET languages also run on any .NET runtime.  Additionally, there are tools on every platform which allow you to build projects.  These tools are the same regardless of your choice of runtime.
+The .NET Standard is an API specification common to all implementations of .NET. Implementations of .NET include the .NET Framework, .NET Core, and Mono for Xamarin. Each implementation of .NET includes the following components:
```

Comment:
Can we open up with something about .NET? Or maybe why would you care about this topic? Seems too direct to start with .NET Standard.